examples/raspberry-pi/image-caption-generator-rPi.json

```diff
@@ -0,0 +1,237 @@
+[
+  {
+      "id": "7a8140fa.cdcf2",
+      "type": "tab",
+      "label": "Image Caption Generator",
+      "disabled": false,
```

Comment:
Can you add info/help similar to what we already have for the getting started flows to this flow?